packages/mux-player/src/html.ts

```diff
@@ -113,6 +116,7 @@ export function processBooleanNode(part: Part, value: unknown): boolean {
 }
 
 export function processPart(part: Part, value: unknown): void {
+  // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
NOTE: These and similar eslint disabling cases are due to changes in default linting rules and application after upgrade.